pandas/tests/indexes/test_range.py

```diff
@@ -315,6 +315,16 @@ def test_is_monotonic(self):
         self.assertTrue(index.is_monotonic_increasing)
         self.assertTrue(index.is_monotonic_decreasing)
 
+        index = RangeIndex(2, 1)
+        self.assertTrue(index.is_monotonic)
+        self.assertTrue(index.is_monotonic_increasing)
+        self.assertTrue(index.is_monotonic_decreasing)
+
+        index = RangeIndex(1, 1)
+        self.assertTrue(index.is_monotonic)
+        self.assertTrue(index.is_monotonic_increasing)
+        self.assertTrue(index.is_monotonic_decreasing)
+
```

Comment:
yeah I don't think ``RangeIndex()`` should be permitted, though I seem to remember needing it for unpickling or somesuch.......